infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -622,7 +625,8 @@ http {
       proxy_set_header Connection "";
       chunked_transfer_encoding off;
 
-      proxy_pass http://s3:9000;
+      set $pass http://s3:9000;
```

Comment:
this is the missing bit. 
for some reason nginx when encounters `proxy_pass http://s3` like directives would try to resolve those at startup time.
But if you declare a variable, parsing would be delayed (?) or until this location was hit.

Other services were already using this approach

Reply:
Yes, I remember @Eijebong fixed something like this for us sometime back. Thanks for fixing here Yarik!

Reply:
Yup, guess I missed this one or it was added later.

Reply:
yeah, i added it later probably, so messed it up. Thanks!